lib/diagnostic.js

```diff
@@ -1,35 +1,58 @@
 'use strict'
 
 // Reuse functions wherever possible.
-
+let paragraph = "the quick brown fox jumped over the fence the quick brown fox jumped over"
+let normalArray = []
 // 1. Write a function that takes a string argument
 // splits it into an array of normalized words
 // (uppercase strings without punctuation)
 // and returns that array.
-const normalizeWords = function (/* */) {
-  /* your response here */
-}
+const words = paragraph.split(" ")
+const normalizeWords = function (wordArray) {
+  for (var i=0; i < wordArray.length; i++) {
+    normalArray[i] = wordArray[i].toUpperCase()
+  }
+};
```

Comment:
Looks good! I think you may have forgotten to get rid of the punctuation however